app/src/main/java/com/sunasterisk/fooddaily/ui/adapter/FoodAdapter.kt

```diff
@@ -18,18 +19,21 @@ class FoodAdapter(
             Constants.FOOD_TYPE_DAILY_MENU -> {
                 DailyMenuViewHolder(
                     LayoutInflater.from(parent.context)
-                        .inflate(R.layout.item_daily_menu_food, parent, false)
+                        .inflate(R.layout.item_daily_menu_food, parent, false),
+                    onClickListener
```

Comment:
Lặp lại code inflater quá
Tách hằng số Type của ViewHolder ra một class mới 
```suggestion
          val inflater = LayoutInflater.from(parent.context)
          return when(viewType){ 
              FoodTypes.ITEM_DAILY_MENU -> 
              FoodTypes.ITEM_DAILY_MENU -> 
              FoodTypes.ITEM_DAILY_MENU -> 
          }
```